src/main/resources/xsd/api_v8.xsd

```diff
@@ -1364,4 +1364,39 @@
         </xsd:restriction>
     </xsd:simpleType>
 
+    <xsd:element name="tag" type="tag" />
+
+    <xsd:complexType name="tag">
+        <xsd:sequence>
+            <xsd:choice>
+                <xsd:element name="public-mailbox-tag" type="public-mailbox-tag">
+                    <xsd:annotation>
+                        <xsd:documentation>
+                            A tag which indicates that the corresponding account
+                            for the provided personal-identification-number should
+                            receive public mail through Digipost
+                        </xsd:documentation>
+                    </xsd:annotation>
+                </xsd:element>
+            </xsd:choice>
+        </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:complexType name="public-mailbox-tag">
+        <xsd:sequence>
+            <xsd:element name="personal-identification-number" minOccurs="0" maxOccurs="1" nillable="false">
```

Comment:
Should probably make the element required by setting `minOccurs` to 1.
Otherwise, for example, the following instance is valid:
```
<?xml version="1.0" encoding="UTF-8"?>
<p:tag xmlns:p="http://api.digipost.no/schema/v8"
    xmlns:p1="http://api.digipost.no/schema/datatypes"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://api.digipost.no/schema/v8 api_v8.xsd ">
    <p:public-mailbox-tag>
    </p:public-mailbox-tag>
</p:tag>
```